src/absenceTypes/absenceTypes.test.ts

```diff
@@ -3,18 +3,21 @@ import { AbsenceType } from './types';
 import axios from 'axios';
 import AxiosMockAdapter from 'axios-mock-adapter';
 import RequestParams from '../utils/requestParams/requestParams';
+import { ApiResponseOnSuccess } from '../utils/response/apiResponse';
 
 describe('AbsenceTypes', () => {
   var absenceTypes: AbsenceTypes = new AbsenceTypes({});
   var readPath: string = `${absenceTypes.getResourcePath()}/read`;
   var mock = new AxiosMockAdapter(axios);
   var result: Promise<AbsenceType[]> | null;
   var resultSingle: Promise<AbsenceType> | null;
+  var resultRaw: Promise<ApiResponseOnSuccess<AbsenceType[]>> | null;
 
   afterEach(() => {
     mock.reset();
     result = null;
     resultSingle = null;
+    resultRaw = null;
```

Comment:
I would remove those vars instead of adding another one, but maybe it also makes sense to do that in a separate PR for all the endpoints at once

Reply:
yes, we will change this in a separate PR